example/package.json

```diff
@@ -18,16 +18,17 @@
   "dependencies": {
     "@react-native-async-storage/async-storage": "^2.2.0",
     "@react-native-clipboard/clipboard": "^1.16.0",
+    "@react-native/new-app-screen": "^0.82.1",
```

Comment:
where is this being used?

Reply:
Good call, that's in RN template but since we're not using it in our app, we can exclude it.